### ISSUE TITLE
Quantity not updated according to recipe serving

### ISSUE THREAD

**Issue by franckleveque** [felixschndr/mealie-bring-api#15]
Hi,

Thank you for this great project!

I’ve encountered an issue when using it to transfer ingredients. Here’s the behavior I observed:

I start with a recipe configured for 1 serving and then increase the number of servings to 4 in mealie.
While the quantities appear updated correctly in the recipe view, when I use the recipe action to transfer the ingredients, the transferred quantities still correspond to 1 serving instead of 4.

Steps to reproduce:

1. Set up a recipe with quantities for 1 serving.
2. Change the number of servings to a larger value (e.g., 4).
3. Use the recipe action to send the ingredient list through the Bring API intermediary server.
4. Check the transferred quantities in Bring – they do not reflect the updated serving size.
5. It seems the intermediary server is not picking up the adjusted quantities. 
 
Let me know if you need further details or testing!

**Comment by felixschndr** [comment]
https://github.com/mealie-recipes/mealie/discussions/4737

Will open if there is some movement on this, until then unfortunately I can't do anything about this.

**Comment by franckleveque** [comment]
Ok thanks for the update. I will follow the mealie discussion.

Le mar. 17 déc. 2024, 14:38, Felix Schneider ***@***.***> a
écrit :

**Comment by felixschndr** [comment]
See https://github.com/felixschndr/mealie-bring-api/pull/17

**Comment by felixschndr** [comment]
FYI: the fields are now added into Mealie (see https://github.com/mealie-recipes/mealie/pull/4952). When the new version is released I will add the functionality to take the number of portions into account

**Comment by Rikshy** [comment]
I have version 2.5 installed and created a test link _http://${slug}/${yieldText}/${servings}/${yieldQuantity}_ and this is what I get when calling the action: 

![Image](https://github.com/user-attachments/assets/38646e7f-1bd6-43c1-8921-a4fe74f16739)

did i miss something?

**Comment by felixschndr** [comment]
The change was merged into mealie after the release of 2.5.0, we'll have to wait for the new release. When it is public I will update this application accordingly asap. Since the releases come quite often I guess it's only a matter of a few days to 2 weeks

**Comment by Rikshy** [comment]
yea sorry I clicked on the release "commits" at the top and thought I will see the commits contained by the release. I was mistaken...

**Comment by felixschndr** [comment]
Waiting on https://github.com/mealie-recipes/mealie/discussions/3610#discussioncomment-12241653 at the moment

**Comment by felixschndr** [comment]
I just opened a PR for this in the mealie codebase https://github.com/mealie-recipes/mealie/pull/5340

**Comment by felixschndr** [comment]
Current status: https://github.com/mealie-recipes/mealie/pull/5340#pullrequestreview-2866376710
